### PR TITLE
Use webpack for styles in dev.

### PR DESCRIPTION
**Pull request by awong-dev**
Fixes `npm run watch` so the styles are kept up to date with the latest changes.

### CODE REVIEW

File: _sass/_va.scss [hunk 1]
```diff
@@ -332,7 +332,7 @@ a:hover {text-decoration: underline; background: rgba(0,0,0,.1);}
 }
 
 html.no-touch .banner {
-  background: url(../img/design/banner.jpg) 50% 60% no-repeat;
+//  background: url(../img/design/banner.jpg) 50% 60% no-repeat;
```
Comment (awong-dev) [inline _sass/_va.scss]: This actually just looks like broken CSS in general.  @akainic @emilyville thoughts?

**Comment by joesoftware** [comment]
LGTM from a webpack standpoint. Looking for comments from others on the CSS potential error.

File: _sass/_va.scss (hunk 1 above)
Comment (akainic) [inline _sass/_va.scss]: I don't see this style even being applied anywhere (can't find an instance of the class `banner` being applied).

Comment (awong-dev) [inline _sass/_va.scss]: I'm going to take this as okay as a change for now then and commit.

Comment (akainic) [inline _sass/_va.scss]: Yep, sounds good

**Comment by webinista** [comment]
Closing in light of our super-duper revamp.

**Pull request closed without merging**